test/mock-values-server/server.py

```
@@ -0,0 +1,41 @@
+            region = params.get("region", [""])[0]
+            values = INSTANCES_BY_REGION.get(region, [])
+        else:
+            user = params.get("user", [""])[0]
+            values = ALLOWED_VALUES.get(user, [])
+
+        self.send_response(200)
+        self.send_header("Content-Type", "application/json")
+        self.end_headers()
+        self.wfile.write(json.dumps(values).encode())
+
+    def log_message(self, format, *args):
```

Ruff (0.15.12)

[error] Function argument `format` is shadowing a Python builtin

(A002)